runtime/binding-tls/src/main/java/io/aklivity/zilla/runtime/binding/tls/internal/config/TlsBindingConfig.java

```diff
@@ -91,12 +92,13 @@ public TlsBindingConfig(
     public void init(
         TlsConfiguration config,
         VaultHandler vault,
-        SecureRandom random)
+        SecureRandom random,
+        TlsEventContext event)
     {
+        this.verifier = new TlsKeyPairVerifier(event);
```

Comment:
If `verifier` is only used locally in this method, then we don't need to store it in a field.
If it is needed by a method called by this method, such as `newKeys` then please pass it as a parameter, instead of introducing more global state as a field.